### ISSUE TITLE
Ninjas can have lavaland ghost role targets

### ISSUE THREAD

**Issue by nazitra** [yogstation13/yogstation#2617]
No, really, I saw a ninja have aashwalker as a target. It might have worked out if they'd made it to the station. They didn't. Ninja had no idea the target was an ashwalker, spent the round looking for someone who 'didn't exist'.

(And lifebringers/golems/etc would also make really shitty protect/kill targets.)

**Comment by Time-Green** [comment]
Actually had this happen aswell, didn't bother with the assasination since ashwalkers tend to be self-destructivr

**Comment by drderp3635** [comment]
Its impossible for steal brain though often times

**Comment by nazitra** [comment]
Yeah, I'd say steal brain/protect for most lavaland ghost roles is near impossible and I'd be annoyed.

(While you're at it, maybe fix the steal research bug?)
